packages/core/services/google-maps-types.ts

```diff
@@ -634,3 +634,65 @@ export interface MapRestriction {
   latLngBounds: LatLngBounds | LatLngBoundsLiteral;
   strictBounds?: boolean;
 }
+
+export interface Geocoder {
```

Comment:
All types that the user will access, such as `GeocoderRequest`, `GecoderResult` and all their dependent types must be exported in `src/packages/core/map-types.ts`